server.py

```diff
@@ -148,25 +188,48 @@ def tick(self):
         #     game.player0.Send({"action":"yourturn", "torf":True if self.games[index].turn==0 else False})
         #     index+=1
         self.Pump()
+
 class Game:
     def __init__(self, player0, currentIndex):
         # whose turn (1 or 0)
-        self.turn = 0
+        self.turn = 0   # Refers to player0 or player1.
+                        # 0 and 1 is too ambiguous.
+                        # Set the value to an enumerated value.
+
         #owner map
-        self.owner=[[False for x in range(6)] for y in range(6)]
-        # Seven lines in each direction to make a six by six grid.
-        self.boardh = [[False for x in range(6)] for y in range(7)]
-        self.boardv = [[False for x in range(7)] for y in range(6)]
+        self.owner: list = [[False for x in range(6)] for y in range(6)]
+            #   Why are we making a 6x6 matrix full of False values?
```

Comment:
Whoops, thought I removed/commented out this stuff. 100% just a holdover from the demo code.